src/components/program/ProgramItem.tsx

```diff
@@ -1,7 +1,11 @@
 import type { TProgramItem } from '../../types/TProgramItem';
```

Comment:
Could you check the framer motion library?
There they have conditionall rendering with animation.
Also, I think, it would be better to show the content but still keep the time and Heading. Basically, just extend the card by its height.